agreement_serviceprofile/models/agreement.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Agreement(models.Model):
+    _inherit = 'agreement'
+
+    serviceprofile_ids = fields.One2many('agreement.serviceprofile',
+                                         'agreement_id',
+                                         default='_set_service_ids',
```

Comment:
@osi-scampbell There is no need for a default value. When creating a service profile from the agreement, the views (tree and form) should not show the agreement field of the service profile. This is the exact same behavior between sale order and sale order line.